service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NoticeOfProceedingsIssuedEventHandler.java

```diff
@@ -26,12 +28,19 @@ public void notifyAllocatedJudgeOfIssuedStandardDirectionsOrder(NoticeOfProceedi
         EventData eventData = new EventData(event);
         CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
 
-        AllocatedJudgeTemplateForNoticeOfProceedings parameters = noticeOfProceedingsEmailContentProvider
+        if (hasAllocatedJudgeEmail(caseData.getNoticeOfProceedings())) {
+            AllocatedJudgeTemplateForNoticeOfProceedings parameters = noticeOfProceedingsEmailContentProvider
                 .buildAllocatedJudgeNotification(eventData.getCaseDetails());
 
-        String email = caseData.getNoticeOfProceedings().getJudgeAndLegalAdvisor().getJudgeEmailAddress();
+            String email = caseData.getNoticeOfProceedings().getJudgeAndLegalAdvisor().getJudgeEmailAddress();
 
-        notificationService.sendEmail(NOTICE_OF_PROCEEDINGS_ISSUED_JUDGE_TEMPLATE, email, parameters,
+            notificationService.sendEmail(NOTICE_OF_PROCEEDINGS_ISSUED_JUDGE_TEMPLATE, email, parameters,
                 eventData.getReference());
+        }
+    }
+
+    private boolean hasAllocatedJudgeEmail(NoticeOfProceedings noticeOfProceedings) {
+        return isNotEmpty(noticeOfProceedings.getJudgeAndLegalAdvisor())
```

Comment:
could change this to getJudgeEmail also instead as it doesn't always pull from allocated judge but from temp or allocated